config/acme/machines/template.st_archive

```diff
@@ -48,12 +48,36 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
     parser.add_argument("--caseroot", default=os.getcwd(),
                         help="Case directory to build")
 
-    CIME.utils.parse_args_and_handle_standard_logging_options(args, parser)
+    parser.add_argument("--last-date", default=None,
+                        help="WARNING: This option with --force-move may corrupt your run directory! Use at your own risk! "
+                        "Last simulation date to archive, specified as 'Year-Month-Day'. "
+                        "Year must be specified with 4 digits, while month and day can be specified without zero padding. "
+                        "'0003-11-4' would archive at most files for the simulated year 3, month 11, day 4."
+                        "This option implies --copy-only unless --force-move is specified ")
+
+    parser.add_argument("--no-incomplete-logs", default=False, action="store_true",
+                        help="Whether to archive logs which have been completed or not")
+
+    parser.add_argument("--copy-only", default=False, action="store_true",
+                        help="Copy instead of move the files to be archived")
+
+    parser.add_argument("--force-move", default=False, action="store_true",
+                        help="Move the files even if it's unsafe to do so")
+
+    args = parser.parse_args()
+
+    CIME.utils.handle_standard_logging_options(args)
```

Comment:
I think this needs to be changed to parse_args_and_handle_standard_logging_options